src/components/wizards/create-vm/CreateVMWizard.js

```diff
@@ -0,0 +1,366 @@
+import React from 'react';
+import { Form, FormControl, Col, ButtonGroup, DropdownButton, MenuItem, Checkbox, FormGroup, ControlLabel, HelpBlock, Wizard } from 'patternfly-react';
+import './CreateVMWizard.css';
+import * as _ from 'lodash';
+import { StorageTable } from './StorageTable';
+
+const defaultState = {
+  activeStepIndex: 0,
+  basicSettings: {},
+  wizardValid: false
+}
+
+export class CreateVMWizard extends React.Component {
+
+    constructor(props){
+      super(props);
+      this.state = defaultState;
+    }
+  
+    onFormChange = (newValue, target) => {
+      let valid = null;
```

Comment:
To avoid confusion with `wizardValid` being a boolean, can we call this `validMsg` or similar?

Reply:
renamed to validMsg